locales/de.json

```diff
@@ -1,47 +1,109 @@
 {
   "aare_valley_label": "Aaretal",
+  "access": "",
```

Comment:
I feel bad having both application translations and data translations in the same file.
Have you considered using i18next namespaces? It looks like designed exactly for our use-case: https://www.i18next.com/principles/namespaces

Concretly the data translations would be put in their own json and will be loaded by an extra fetch.

Reply:
That's a very good idea